src/_base.scss

```diff
@@ -32,6 +32,13 @@
   background-color: var(--bg-warning);
 }
 
+%imageMeaningfulAlt {
+  background: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="400px" height="80px" viewBox="0 0 400 80"><rect class="box" x="4" y="4" width="392" height="72" rx="12" ry="12" fill="%23ffffd8" stroke-width="0.4rem" stroke="%23ff6" /><g fill="%23856404" font-size="1.2rem" font-weight="bold" font-family="Verdana, Geneva, Tahoma, sans-serif"><text x="15" y="35">WARNING: Alt text must describe</text><text x="15" y="60">the content of the image.</text></g></svg>') bottom center no-repeat;
+  border: 0.4rem solid var(--border-warning);
+  min-width: 400px;
+  padding-bottom: 80px;
```

Comment:
Could we use the `var(--space-x)` variables here?

Reply:
I'll move the width and padding bottom to the mixin as they match the size of the image. Because of SVG text limitations, each line has to be specified independently, so the mixin could be something like this:

```
generateImage(type, width, height, line1, line2, line3)
```

Reply:
I finally decided to change the generateImage mixin so it takes multiple parameters:

- typeError: the type of the error (warning or error)
- width: the width of the background image (will be applied as min-width to the element)
- lines...: the rest of the parameters will be added to an array of lines.

All parameters are optional. If no parameters are passed, it will generate a warning background image 400px-wide with the message "CAUTION: Unidentified issue."

I added a couple of examples (warning for the meaningful alt in images, and error for the embed inside button).